bindings/binding.cpp.tmpl

```diff
@@ -0,0 +1,16 @@
+// This file was autogenerated by the Chimera build.
+
+// Include binding definitions for each built-in language.
+@BINDING_INCLUDES_LIST@
+
+// Set the default binding.
+std::string chimera::binding::DEFAULT_NAME = "boost_python";
+
+// Create the built-in definitions map.
+std::map<std::string, chimera::binding::Definition> chimera::binding::DEFINITIONS;
+
+// Define loader function for all bindings.
+void chimera::binding::initializeBuiltinBindings()
+{
+    @BINDING_REGISTRATIONS_LIST@
+}
```

Comment:
We can eliminate this function by generating an initialization list for `chimera::binding::DEFINITIONS`. I believe that simply requires changing `binding.cpp.tmpl` to contain:
```c++
std::map<std::string, chimera::binding::Definition> chimera::binding::DEFINITIONS {
@BINDING_REGISTRATIONS_LIST@
};
```
And modifying `bindings/CMakeLists.txt` to contain:
```c++
list(APPEND BINDING_REGISTRATIONS_LIST
  { \"${binding_NAME}\", ${binding_NAME}_DEFINITION },
```
You might need to wrap the string in an `std::string`, no warranty, etc.